src/platform/nxp/rt/rt1060/args.gni

```diff
@@ -32,6 +32,9 @@ chip_ble_project_config_include = "<CHIPProjectConfig.h>"
 
 chip_build_tests = false
 
+# Disable TCP endpoint.
```

Comment:
nit: this comment seems obvious (repeats the code below). Could we update the comment to help reading the code better instead? Like why is it disabled (flash size, tcp support in the chipset, something else?)